test/expected/drop_rename_hypertable.out

```diff
@@ -317,32 +317,178 @@ Check constraints:
+ 1257987600000000000 | dev1      |      1.5 |        1 |          | 
+ 1257987600000000000 | dev1      |      1.5 |        2 |          | 
+ 1257894000000000000 | dev2      |      1.5 |        1 |          | 
+ 1257894000000000000 | dev2      |      1.5 |        2 |          | 
+(12 rows)
 
 SELECT * FROM _timescaledb_catalog.hypertable;
- id | schema_name |   table_name   | associated_schema_name | associated_table_prefix | num_dimensions 
-----+-------------+----------------+------------------------+-------------------------+----------------
-  1 | public      | two_Partitions | _timescaledb_internal  | _hyper_1                |              2
+ id | schema_name | table_name | associated_schema_name | associated_table_prefix | num_dimensions 
+----+-------------+------------+------------------------+-------------------------+----------------
+  1 | newschema   | newname    | _timescaledb_internal  | _hyper_1                |              2
 (1 row)
 
-DROP TABLE "two_Partitions" CASCADE;
+SELECT * FROM _timescaledb_catalog.hypertable_index;
+ hypertable_id | main_schema_name |              main_index_name              |                                                              definition                                                               
+---------------+------------------+-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------
+             1 | newschema        | two_Partitions_device_id_timeCustom_idx   | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree (device_id, "timeCustom" DESC NULLS LAST) WHERE (device_id IS NOT NULL)
+             1 | newschema        | two_Partitions_timeCustom_series_0_idx    | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC NULLS LAST, series_0) WHERE (series_0 IS NOT NULL)
+             1 | newschema        | two_Partitions_timeCustom_series_1_idx    | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC NULLS LAST, series_1) WHERE (series_1 IS NOT NULL)
+             1 | newschema        | two_Partitions_timeCustom_series_2_idx    | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC NULLS LAST, series_2) WHERE (series_2 IS NOT NULL)
+             1 | newschema        | two_Partitions_timeCustom_series_bool_idx | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC NULLS LAST, series_bool) WHERE (series_bool IS NOT NULL)
+             1 | newschema        | two_Partitions_timeCustom_device_id_idx   | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC NULLS LAST, device_id)
+             1 | newschema        | two_Partitions_timeCustom_idx             | CREATE INDEX /*INDEX_NAME*/ ON /*TABLE_NAME*/ USING btree ("timeCustom" DESC)
+(7 rows)
+
+SELECT * FROM _timescaledb_catalog.chunk_index;
```

Comment:
Don't we only want to check schema_name here maybe `SELECT * FROM _timescaledb_catalog.chunk_index WHERE schema_name <> 'newschema'`